tokenizers/src/models/unigram/model.rs

```diff
@@ -53,8 +53,8 @@ impl Clone for Unigram {
 
 impl std::fmt::Debug for Unigram {
     fn fmt(&self, fmt: &mut std::fmt::Formatter) -> std::fmt::Result {
-        fmt.debug_struct("BPE")
-            .field("vocab", &self.vocab.len())
+        fmt.debug_struct("Unigram")
+            .field("vocab", &self.vocab)
```

Comment:
That's quite a large log. There's only vocab.len() in BPE for instance. Is there a reason to output the whole vocab here ?

Reply:
Oops my bad, needed it for debug, but we should keep the `len` yes!